readthedocs/doc_builder/templates/doc_builder/conf.py.tmpl

```diff
@@ -143,4 +143,4 @@ else:
     extensions = ["readthedocs_ext.readthedocs"]
 
 # Build the json artifacts with the html build step
-rtd_generate_json_artifacts = {{ generate_json_artifacts }}
+rtd_generate_json_artifacts = True
```

Comment:
I didn't find anything where we are using this, but removing it may break other things maybe. I don't know.

Reply:
Removed and opened another PR at https://github.com/rtfd/readthedocs-sphinx-ext/pull/50